pkg/database/facade/inserter.go

```diff
@@ -16,15 +16,15 @@ type DBInserter interface {
 		clusterDeletedTs sql.NullString,
 		data []byte,
 	) *sqlbuilder.InsertBuilder
-	UrlInserter(uuid, url, containerName string) *sqlbuilder.InsertBuilder
+	UrlInserter(uuid, url, containerName, jsonPath string) *sqlbuilder.InsertBuilder
```

Comment:
Need to make sure that a `null` doesn't get written to the database as the string "null".

Reply:
since `UrlInserter` accepts a jsonPath as a string, the closest thing to `null` that go will let you pass is empty string. I tested with the following logging config (note that there is not `LOG_URL_JSONPATH` defined)

```
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: kubearchive-logging
  namespace: kubearchive
data:
  # NOTE: CONTAINER_NAME is provided at URL generation time by KubeArchive.
  POD_ID: "cel:metadata.uid"
  LOG_URL: "https://localhost:8089/services/search/jobs/export?search=search%20%2A%20%7C%20spath%20%22kubernetes.pod_id%22%20%7C%20search%20%22kubernetes.pod_id%22%3D%22{POD_ID}%22%20%7C%20spath%20%22kubernetes.container_name%22%20%7C%20search%20%22kubernetes.container_name%22%3D%22{CONTAINER_NAME}%22%20%7C%20sort%20time%20%7C%20table%20%22message%22&output_mode=json"
```
when I look at the `log_url` table, I see that the rows have an empty string for the `json_path` column, not `"null"`